clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -11065,7 +11065,7 @@ void LinkerWrapper::ConstructJob(Compilation &C, const JobAction &JA,
   ArgStringList CmdArgs;
 
   // Pass the CUDA path to the linker wrapper tool.
-  for (Action::OffloadKind Kind : {Action::OFK_Cuda, Action::OFK_OpenMP}) {
+  for (Action::OffloadKind Kind : {Action::OFK_Cuda, Action::OFK_OpenMP, Action::OFK_SYCL}) {
```

Comment:
is there a test we can update to verify that `--cuda-path` is being passed from the driver?

Reply:
Good point. Comping soon!!

Reply:
Added in the latest commit